test/vm.install

```diff
@@ -43,8 +43,20 @@ if [ -d /var/tmp/debian ]; then
 # install rpms
 elif [ -e /var/tmp/*.rpm ]; then
     rpm -i --verbose /var/tmp/*.rpm
-fi
+elif [ -d /var/tmp/arch ]; then
+    # build source package
+    cd /var/tmp
+    TAR=$(ls cockpit-machines-*.tar.xz)
+    VERSION="${TAR#cockpit-machines-}"
+    VERSION="${VERSION%.tar.xz}"
 
+    cp arch/PKGBUILD .
+    sed -i "s/VERSION/$VERSION/" PKGBUILD
+    sed -i "s/SOURCE/$TAR/" PKGBUILD
```

Comment:
Unimportant, but for the future: put both s// sed commands into one CLI call (semicolon separated).